docs/starters.yml

```diff
@@ -1647,3 +1647,15 @@
     - Comes with React Helmet for adding site meta tags
     - Add Disqus
     - Nice Pagination
+- url: https://master.d2f5ek3dnwfe9v.amplifyapp.com/
+  repo: https://github.com/dabit3/gatsby-auth-starter-aws-amplify
+  description: This Gatsby starter uses AWS Amplify to implement authentication flow for signing up signing in users as well as protected client side routing.
+  tags:
+    - aws
+    - authentication
+    - amplify
+  features:
```

Comment:
Feel free to ignore these comments below as it's just a suggestion to rephrase a few things

Reply:
This is the updated to tag so will wait for @DSchau

Reply:
lets do this
```
features:
    - AWS Amplify
    - Full authentication workflow
    - Registration form
    - Signup form
    - User sign in
```